ui.frontend/src/view/FormPanel.js

```diff
@@ -150,5 +150,8 @@ export default class FormPanel extends FormFieldBase {
         }
         return resultIndex + 1;
     }
+    getRepeatableRootElement(childView){
+      return childView.element.parentElement;
```

Comment:
why it will always be parentElement? we should be using element.closest here and then find something that is unique to detecting repeatable element.

Reply:
We cannot use element.closest here because it could be any type of layout component.
(Based on the previous implementation, we are only obtaining the parent element https://github.com/adobe/aem-core-forms-components/blob/master/ui.frontend/src/view/InstanceManager.js#L157)

Reply:
I see we are not supporting repeatable inside repeatable. is it a known limitation? cc @sakshi-arora1

Reply:
**we are supporting repeatable inside repeatable**. The private function doesn't support it till date, as that private function usage was limited till now. If intend to use it in this case, it will need to be enhanced.

Reply:
done @sakshi-arora1